integration-tests/src/test/resources/integration-test.anchor-config.yaml

```diff
@@ -143,6 +143,8 @@ data-spring-jdbc-sqlite:
   spring.jpa.hibernate.show_sql: false
   spring.datasource.url: jdbc:sqlite:anchor-proxy.db
   spring.datasource.driver-class-name: org.sqlite.JDBC
+  spring.datasource.max-active: 2
```

Comment:
Same as above. 

If this is failing because of the database locking issue, we can use H2 for now and fix this post config refactoring.

Reply:
I didn't manage to setup a common H2 database to be shared across multiple services.